cmd/protoc-gen-cpp-tableau-loader/embed/util.pc.cc

```diff
@@ -58,16 +58,10 @@ const std::string& Format2Ext(Format fmt) {
   }
 }
 
-bool JSON2Message(const std::string& json, google::protobuf::Message& msg,
-                  std::shared_ptr<const MessagerOptions> options /* = nullptr */) {
-  google::protobuf::util::Status status;
-  if (options != nullptr) {
-    google::protobuf::util::JsonParseOptions parse_options;
-    parse_options.ignore_unknown_fields = options->ignore_unknown_fields.value_or(false);
-    status = google::protobuf::util::JsonStringToMessage(json, &msg, parse_options);
-  } else {
-    status = google::protobuf::util::JsonStringToMessage(json, &msg);
-  }
+bool JSON2Message(const std::string& json, google::protobuf::Message& msg, bool ignore_unknown_fields) {
+  google::protobuf::util::JsonParseOptions parse_options;
+  parse_options.ignore_unknown_fields = ignore_unknown_fields;
```

Comment:
For extensibility and func signature stability, a better solution is to pass `google::protobuf::util::JsonParseOptions` param.

See https://protobuf.dev/reference/cpp/api-docs/google.protobuf.util.json_util/#JsonStringToMessage.details

Example:
```cpp
bool JSON2Message(const std::string& json, google::protobuf::Message& msg, const google::protobuf::util::JsonParseOptions& options)
```